clusters/app.ci/openshift/admin_image-puller_rolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: image-puller
+  namespace: openshift
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:image-puller
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: system:authenticated
```

Comment:
I don't think this is correct, these should be imported via an authenticated pull secret and then pulling from a local registry should add auth if it ends up being a pull-through pull

Reply:
/hold
Maybe talk with oleg to figure out why the pull-through auth isn't working after the auth apparently worked during import